### ISSUE TITLE
I can't open Cheat Engine 7.4 w10

### ISSUE THREAD

**Issue by oslini23** [cheat-engine/cheat-engine#2434]
windows defender disabled
no windows error
click on the app and nothing happens

**Comment by cheat-engine** [comment]
is realtime protection also disabled ?
And uninstall vanguard, badlion and any other anti cheat

**Comment by oslini23** [comment]
yes it is disabled realtime protection (to install and to open the application)
only have windows defender

strange not even a version 7 of the cheat engine opens. I've tried them all
only version 6.4 opens correctly
